docs/src/Checkbox.doc.js

```diff
@@ -74,6 +74,13 @@ a checkbox and its indeterminism are independent.`,
         name: 'onClick',
         type: `({ event: SyntheticInputEvent<HTMLInputElement>, checked: boolean }) => void`,
       },
+      {
+        name: 'ref',
+        type: "React.Ref<'div'>",
```

Comment:
@AlbertCarreras can we change this to `"React.Ref<'input'>"`?

Reply:
DONE